srcs/error/print_error.c

```diff
@@ -24,7 +24,7 @@ void					print_error_msg(t_error_id id)
 	{
 		if (id == errors[u].id)
 		{
-			ft_putendl(errors[u].msg);
+			ft_putendl_fd(errors[u].msg, 2);
```

Comment:
Nah.
What is the problem?
I did it well.
https://github.com/42shpimanmls/42sh/pull/205/files#diff-dca45fdca04f2864d393a768c0ab2192L28